MapManager/MapManager_Reverse.cpp

```diff
@@ -1,6 +1,41 @@
 #include "MapManager.h"
 
-//���Ӳ���ɵ�ͼ�ķ�ת
+int dir[8][2]={{0,1},{0,-1},{1,0},{-1,0},{1,1},{1,-1},{-1,1},{-1,-1}};
+
+//判断在棋盘内
+bool In_Range(int x,int y)
+{
+  return (1<=x && x<=8 && 1<=y && y<=8);
```

Comment:
The size of the map might be changed to 10*10.
You should use macro "ColNumber" and "RowNumber" instead of a specific number in any cases.

Reply:
confirm.